src/rebar_prv_dialyzer.erl

```diff
@@ -193,9 +194,20 @@ do_update_proj_plt(State, Plt, Output) ->
     end.
 
 proj_plt_files(State) ->
-    BasePltApps = base_plt_apps(State),
+    {BasePltApps, BasePltMods} =
+        case is_update_base_plt_active(State) of
+            false ->
+                {ok, BaseFiles} = plt_files(get_base_plt(State)),
+                BaseMods =
+                    lists:map(fun(File) ->
+                                      list_to_atom(filename:basename(File, code:objfile_extension()))
+                              end, BaseFiles),
+                {[], BaseMods};
+            true  ->
+                {base_plt_apps(State), get_config(State, base_plt_mods, [])}
+        end,
+
     PltApps = get_config(State, plt_extra_apps, []) ++ BasePltApps,
-    BasePltMods = get_config(State, base_plt_mods, []),
```

Comment:
Unfortunately that logic is not consistent. To be consistent we need to use the value on this line for base plt modules.